packages/tasks/src/model-libraries.ts

```diff
@@ -783,6 +783,21 @@ export const MODEL_LIBRARIES_UI_ELEMENTS = {
 		snippets: snippets.pxia,
 		filter: false,
 	},
+	"pruna-ai": {
+		prettyLabel: "Pruna AI",
+		repoName: "Pruna AI",
+		repoUrl: "https://github.com/Pruna-AI/pruna-ai",
+		snippets: snippets.pruna,
+		docsUrl: "https://docs.pruna.ai",
+		filter: false,
+	},
+	"pruna_pro-ai": {
+		prettyLabel: "Pruna AI (pro)",
+		repoName: "Pruna AI (pro)",
+		repoUrl: "https://github.com/Pruna-AI/pruna-ai",
+		snippets: snippets.pruna_pro,
+		docsUrl: "https://docs.pruna.ai",
+	},
```

Comment:
not a fan of having two ≠ libraries. IMO this should always be Pruna (and you can prompt users to upgrade to a "pro" version from inside your library/error-messages/documentation/etc)